src/Scouting.elm

```diff
@@ -55,32 +61,91 @@ update msg model =
         TeamInput input ->
             { model
                 | team = String.toInt input
-                , driverStation = stationIndex (String.toInt input) model.match
+                , driverStation = stationName (String.toInt input) model.match
             }
 
         MatchInput input ->
             { model
                 | match = String.toInt input
-                , driverStation = stationIndex model.team (String.toInt input)
+                , driverStation = stationName model.team (String.toInt input)
             }
 
         Start ->
             { model
-                | isStarted = not model.isStarted
-                , driverStation = stationIndex model.team model.match
+                | isStarted = switchButtonState model
+                , driverStation = stationName model.team model.match
             }
 
 
-startButton : Bool -> Html.Html Msg
-startButton isStarted =
-    Html.div []
-        [ Html.button [ onClick Start ]
-            [ Html.text <|
-                if isStarted then
-                    "Started"
+ifCorrectInput : Model -> ButtonState
+ifCorrectInput model =
+    let
+        modelStr =
+            strModel model
+
+        isStation string =
+            model.driverStation == string
+    in
+    if modelStr.scouter == "" then
+        TriedPush
+
+    else if modelStr.team == "" then
+        TriedPush
+
+    else if modelStr.match == "" then
+        TriedPush
+
+    else if isStation " " || isStation "  " then
+        TriedPush
+
+    else
+        Pushed
+
+
+switchButtonState : Model -> ButtonState
+switchButtonState model =
+    if model.isStarted == Pushed then
+        Untouched
+
+    else
+        ifCorrectInput model
+
+
+inputHelpMessage : String -> String
+inputHelpMessage strStation =
+    case strStation of
+        " " ->
+            "Not a match"
+
+        "  " ->
+            "Team not in this match"
+
+        "" ->
+            "Please fill all inputs."
+
+        _ ->
+            ""
+
+
+startButton : StrModel -> Html.Html Msg
+startButton model =
+    let
+        ifStarted : String -> String -> Html.Html Msg
+        ifStarted ifYes ifNo =
+            Html.text <|
+                if model.isStarted == Pushed then
+                    ifYes
 
                 else
-                    "Startn't"
+                    ifNo

```

Comment:
I think you'll find it useful to define a helper like
```elm
choose : Bool -> a -> a -> a
choose condition ifTrue ifFalse =
    if condition then
        ifTrue

    else
        ifFalse
```
(This is really just a wrapper around `if` so that we can branch out on a single line - because `elm-format` always splits branches to different lines. If you're familiar with other languages' ternary operators - that's exactly Elm's equivalent: `condition ? ifTrue : ifFalse` is just like `choose condition ifTrue ifFalse`).